### ISSUE TITLE
Flex Module support

### ISSUE THREAD

**Issue by devlfm** [BowlerHatLLC/vscode-as3mxml#390]
Hi @joshtynjala 
It would be nice to have a better handling on flex modules.
I'm missing some command like "> Add flex module" (suggests items based on files that starts with <s|mx:Module  or file-browse), "> Remove flex module" (show current configured modules to select to remove).

The need to create asconfig.module.json for each module that needs to be compiled first, and then setup the tasks is too much to setup manually.

**Comment by devlfm** [comment]
It would be handful to have an option on tasks.json to pass an array of asconfig(s).json, on the property "asconfig", like this:
```
{
    "label": "build-debug-main",
    "type": "actionscript",
    "problemMatcher": [
        "$nextgenas_nomatch"
    ],
    "debug": true
},
{
    "label": "build-debug-modules",
     "type": "actionscript",
     "asconfig": [
         "asconfig.module-a.json",
         "asconfig.module-b.json",
     ]
     "problemMatcher": [
          "$nextgenas_nomatch"
      ],
      "debug": true
},
{
    "label": "build-debug",
    "dependsOrder": "sequence",
    "dependsOn": [
        "build-debug-main",
        "build-debug-modules"
    ],
    "problemMatcher": [
        "$nextgenas_nomatch"
    ],
    "group": {
        "kind": "build",
        "isDefault": true
    }
},
```
or, inside asconfig.json an array for modules perhaps..

One of my projects, have 10 modules, my tasks.json have more than 160lines, and I haven't setup yet the release tasks -.-'

**Comment by joshtynjala** [comment]
When I get around to adding better support for modules, I will probably add a field in asconfig.json where you can specify one or more modules. Then, it will build the main app and the modules all in a single task.

**Comment by devlfm** [comment]
Any news on this?

**Comment by joshtynjala** [comment]
No news at this time.

**Comment by devlfm** [comment]
Hi @joshtynjala, my finger is on the trigger waiting on modules support 😄

**Comment by devlfm** [comment]
🙏🥳🥳🥳
